lib/mix/triplex.ex

```diff
@@ -103,7 +103,7 @@ defmodule Mix.Triplex do
 
     migrated =
       Enum.flat_map(Triplex.all(repo), fn tenant ->
-        opts = Keyword.put(opts, :prefix, tenant)
+        opts = Keyword.put(opts, :prefix, Triplex.to_prefix(tenant))
```

Comment:
Well, I'm not sure that will actually work, because as you can see in the line above we call `Triplex.all`, which returns the tenants already with their prefixes, that way we don't actually need to call `Triplex.to_prefix` on them. But you at least got me thinking, maybe we should rename `Triplex.all` to `Triplex.all_prefixes` or something like that. We need to think of better name convention on the whole project maybe.

Reply:
Or even, `Triplex.all` should be changed so that it takes the prefix off from all items in the return value.

Reply:
I looked at the definition of Triplex.all and it does not appear to add prefixes. https://github.com/ateliware/triplex/blob/c8b3ee21dd12cb6432263db319f36b125752147e/lib/triplex.ex#L276

Reply:
Nope, it does not add, the prefixes are added when the tenant is created, therefor, when `Triplex.all` queries for them, they are already there.

Reply:
Okay, this is probably a MySQL thing then. My tenants table contains the unprefixed tenant names (which does make sense), which is why this fix worked for me.

I am prepping another pull request that adds support for MyXQL, so maybe it would make sense to combine the two PRs (changing this fix to be MySQL-only).